fat32/sdcard.s

```diff
@@ -558,3 +562,24 @@ sdcard_check_alive:
 	jsr deselect
 	plp
 	rts
+
+;-----------------------------------------------------------------------------
+; sdcard_set_fast_mode
+;
+; .A = 0: fast mode off
+; .A = 1: fast reads
+; .A = 2: fast writes
+; .A = 3: fast reads and writes
+;-----------------------------------------------------------------------------
+sdcard_set_fast_mode:
```

Comment:
Something for possible consideration: there could be a situation where for some reason the user's system cannot tolerate fast read or write (e.g. the system is overclocked or we enable a slower SPI bus to a Wi-Fi peripheral). It might be good to have a hard override that prevents setting fast reads or writes.

Reply:
How do you see this working?

For now, the variable gets initialized to 0 at boot (with the most recent push) and the user has to explicitly change it.